### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."<title>

### DISCUSSION THREAD

**Discussion by aroo530** [community/community#43695] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

**Hello,**

I have been using CoPilot for a while now and have previously used the free trial. I switched to the student pack around two months ago.

I have **searched** the forum and all posts say that there is a bug that should have been fixed in November. However, the issue persists. Can anyone tell me how to fix this and if CoPilot has been removed from student packs?

![image](https://user-images.githubusercontent.com/66312772/211258723-4f7b1cf6-4dff-4e46-9aaf-a3f913ff1228.png)
![image](https://user-images.githubusercontent.com/66312772/211258802-4787589c-fb5f-4ebe-977b-fc89705e7b56.png)

Thanks in advance

**Comment by HasGenvts** [comment]
the same issue!
just wait

**Reply by aroo530** [reply]
okay

**Comment by jongwooo** [comment, marked as answer]
Solved: #43634
